shell/config/router.js

```diff
@@ -1,506 +0,0 @@
-        }, {
-          path:      '/c/:cluster/auth/roles/:resource/:id?',
-          component: () => interopDefault(import('../pages/c/_cluster/auth/roles/_resource/_id.vue')),
-          name:      'c-cluster-auth-roles-resource-id'
-        }, {
-          path:      '/c/:cluster/monitoring/monitor/:namespace/:id?',
-          component: () => interopDefault(import('../pages/c/_cluster/monitoring/monitor/_namespace/_id.vue')),
-          name:      'c-cluster-monitoring-monitor-namespace-id'
-        }, {
-          path:      '/c/:cluster/navlinks/:group?',
-          component: () => interopDefault(import('../pages/c/_cluster/navlinks/_group.vue')),
-          name:      'c-cluster-navlinks-group'
-        }, {
-          path:      '/c/:cluster/:product',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/index.vue')),
-          name:      'c-cluster-product'
-        }, {
-          path:      '/c/:cluster/:product/members',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/members/index.vue')),
-          name:      'c-cluster-product-members'
-        }, {
-          path:      '/c/:cluster/:product/namespaces',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/namespaces.vue')),
-          name:      'c-cluster-product-namespaces'
-        }, {
-          path:      '/c/:cluster/:product/projectsnamespaces',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/projectsnamespaces.vue')),
-          name:      'c-cluster-product-projectsnamespaces'
-        }, {
-          path:      '/c/:cluster/:product/:resource',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/index.vue')),
-          name:      'c-cluster-product-resource'
-        }, {
-          path:      '/c/:cluster/:product/:resource/create',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/create.vue')),
-          name:      'c-cluster-product-resource-create'
-        }, {
-          path:      '/c/:cluster/:product/:resource/:id',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/_id.vue')),
-          name:      'c-cluster-product-resource-id'
-        }, {
-          path:      '/c/:cluster/:product/:resource/:namespace/:id?',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/_namespace/_id.vue')),
-          name:      'c-cluster-product-resource-namespace-id'
-        }]
-    }],
-
-  fallback: false
-};
-
-export function createRouter(config) {
-  const base = (config._app && config._app.basePath) || routerOptions.base;
-  const router = new Router({ ...routerOptions, base });
-
-  // TODO: remove in Nuxt 3
-  const originalPush = router.push;
-
-  router.push = function push(location, onComplete = emptyFn, onAbort) {
-    return originalPush.call(this, location, onComplete, onAbort);
-  };
```

Comment:
We only used push with an empty second argument in one place. I updated that location so we can remove this code.